sips/completed/_posts/2016-06-25-trailing-commas.md

```diff
@@ -2,9 +2,10 @@
 layout: sip
 disqus: true
 title: SIP-27 - Trailing Commas
+redirect_from: "/sips/pending/trailing-commas.html"
 
-vote-status: under review
-vote-text: The following proposal needs to be updated, since only the specialized case version (with new lines) has been accepted. For more information, check the <a href="http://docs.scala-lang.org/sips/minutes/sip-20th-september-minutes.html">minutes</a>.
+vote-status: approved
+vote-text: The following proposal has been approved but the spec needs to be updated and the implementation has to be ported from Parsers to Scanner. For more information, check the minutes in November 2017.
```

Comment:
November 2017? Do you mean November 2016? In any case, these minutes do not exist yet, so it's impossible to check this. Perhaps omit this until you have done the minutes?